net-misc/netkit-rsh/files/patches/000_all_sectty.patch

```diff
@@ -0,0 +1,32 @@
+--- netkit-rsh-0.17-pre20000412/rexecd/rexecd.c.sectty	Sat Oct  2 17:50:52 1999
```

Comment:
This is just to be nice IF you can: can you drop the dates and the directory names at the beginning? Change to a/ and b/?


See here: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto. Alright if not though, it's a nit.